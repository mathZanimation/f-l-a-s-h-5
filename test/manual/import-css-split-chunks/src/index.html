<!DOCTYPE html>
<html>
<head>
  <link href="@images/favicons/favicon.svg" rel="icon">
  <link href="./css/style.css" rel="stylesheet">
  <script src="./js/app.js"></script>
</head>
<body>
  <h1>Home 123</h1>
  <div class="lib">.lib</div>
  <div class="app">.app</div>
  <div class="main-a">.main-a</div>
  <div class="main-b">.main-b</div>
</body>
</html>