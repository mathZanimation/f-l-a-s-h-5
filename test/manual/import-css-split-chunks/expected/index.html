<!DOCTYPE html>
<html>
<head>
  <link href="css/style.bundle.css" rel="stylesheet">
  <link href="css/app.bundle.css" rel="stylesheet">
<script src="js/runtime.bundle.js"></script>
<script src="js/589.bundle.js"></script>
<script src="js/540.bundle.js"></script>
<script src="js/313.bundle.js"></script>
<script src="js/631.bundle.js"></script>
<script src="js/app.bundle.js"></script>
</head>
<body>
  <h1>Home</h1>
  <div class="lib">.lib</div>
  <div class="app">.app</div>
  <div class="main-a">.main-a</div>
  <div class="main-b">.main-b</div>
</body>
</html>